overlord/devicestate/devicestate_test.go

```diff
@@ -330,12 +330,29 @@ func (s *deviceMgrSuite) TestDeviceManagerEnsureSeededHappy(c *C) {
 }
 
 func (s *deviceMgrSuite) TestDeviceManagerEnsureBootOkSkippedOnClassic(c *C) {
+	s.bootloader.GetErr = fmt.Errorf("should not be called")
 	release.OnClassic = true
```

Comment:
Shouldn't this use `release.MockOnClassic()`? Also, should we explicitly set the operating mode here to prevent falling in the run mode test condition?

Reply:
there's a MockOnClassic in the common SetUpTest